File: 백준/Bronze/5086. 배수와 약수/배수와 약수.cc
#include <iostream>
using namespace std;

int main()
{
	int num1;
	int num2;

	while (true)
	{
		cin >> num1 >> num2;

		if (num1 == 0 && num2 == 0)
		{
			break;
		}
		
		if (num1 % num2 == 0)
		{
			cout << "multiple" << endl;
		}
		else if (num2 % num1 == 0) {
			cout << "factor" << endl;
		}
		else
		{
			cout << "neither" << endl;
		}
	}
}

File: 백준/Bronze/5086. 배수와 약수/배수와 약수.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        while(true) {
            int num1 = sc.nextInt();
            int num2 = sc.nextInt();

            if (num1 == 0 && num2 == 0) {
                return;
            }

            if (num1 > num2) {
                if (num1 % num2 == 0) {
                    System.out.println("multiple");
                } else {
                    System.out.println("neither");
                }
            } else {
                if (num2 % num1 == 0) {
                    System.out.println("factor");
                } else {
                    System.out.println("neither");
                }
            }
        }
    }
}